cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebViewHelper.java

```diff
@@ -95,6 +99,60 @@ public void run() {
         });
     }
 
+    @TargetApi(11)
```

Comment:
It is for suppress the lint error. Libcocos2d has the minsdk at 10. These methods are available from 11 sdk. I could compile it with the minsdk at 10 and I figured out that meanwhile no one use these methods in a device with 10 api wont be error. 

I cannot try it because I cant simulate it and dont have a device with 10 api. 
So I need to know if the cocos2d library would work on a 10-device if dont use these methods. And in case of YES warn the developer that if he want to use these, he must put his game with a minimum sdk of 11. And in case of NO, know how to discriminate it in old devices.

Im learning yet, and I wanted to contribute with Cocos2d-x. So sorry for the lack of skills. Im looking for a solution for this problem yet. If someone can enlighten me I will be grateful.